pkg/sql/lease.go

```diff
@@ -41,14 +41,20 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/util/timeutil"
 )
 
-// TODO(pmattis): Periodically renew leases for tables that were used recently and
-// for which the lease will expire soon.
-
```

Comment:
you can rename the commit

sql: table lease refresh done asynchronously

Reply:
Done.